### ISSUE TITLE
Pages or directories at the root level that start with a number break the pdf generation

### ISSUE THREAD

**Issue by geealbers** [thegetty/quire#752]
### Before proceeding, check to make sure there isn’t an existing issue for this bug.

- [X] I have searched the existing issues and determined this is a new bug.

### Steps to reproduce

1. Start a new project
2. Rename a directory and/or a markdown file in the `content` directory so that it starts with a number (0–9).
3. run `quire build` and then `quire pdf`

### Actual behavior

An error is thrown and the pdf is not generated

```
Error: Evaluation failed: DOMException: Failed to execute 'querySelector' on 'Element': '#10-intro' is not a valid selector.
```

### Expected behavior

The PDF would generate and no error would be thrown.

Sub-directories and markdown pages within directories can have file names that start with numbers and not break the PDF. So the actual behavior is inconsistent. It's also problematic that the error doesn't happen until the user tries to generate the PDF as this is often late into the development process. Also that the error messaging isn't helpful unless one happens to know `id` selectors can't start with numbers.

### Version numbers

macOS Ventura 13.2.1 (22D68)
Node 18.16.0
Quire CLI 1.0.0-rc.7
Quire 11ty 1.0.0-rc.10

**Comment by Erin-Cecele** [comment]
We have confirmed that the reported issue is a bug. It has been added to our development queue. Stay tuned for updates.

**Comment by Erin-Cecele** [comment]
Our developers are currently working on this issue. Please check back for updates.

**Comment by Erin-Cecele** [comment]
This was fixed in PR #808. Issue is now closed.
